functions/helloworld/test/sample.system.storage.test.js

```diff
@@ -47,6 +47,7 @@ it('helloGCS: should print uploaded message', async () => {
     .toString();
   assert.ok(logs.includes(`File ${gcsFileName} uploaded`));
 });
+// [END functions_storage_system_test]
```

Comment:
We don't want to include the other system tests (deletion, metadata)?

Reply:
Yeah - not including them = shorter sample in the docs. 🙂